app/views/groups/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= stylesheet_link_tag 'groups/index', media: 'all' %>
+
+<div class= 'category-cont'>
+
+<div class='category-header'>
+<%=link_to groups_path do %>
+    <i class="fas fa-arrow-left"></i>
+<%end%>
+
+<h1 class='category-title'>Add Category</h1>
+        <%= link_to destroy_user_session_path, data: {turbo_method: :delete, turbo_confirm: "Are you sure?"} do %>
+    <i class="fas fa-sign-out-alt"></i>
+<% end %>
+</div>
+
+<div class='categories_container'>
+
+<%= form_for @categorie, url: groups_path, method: :post do |f| %>
+    <div class='categorie-body'>
+        <div class='form-group'>
+            <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder:'Name', class:'form-inp' %>            
+        </div>
+
+        <div class='form-group'>
+            <%= f.text_field :icon, autofocus: true, autocomplete: "icon", placeholder:'Icon', class:'form-inp' %>
+        </div>
+    </div>
+<div class='btn-cont'>
+    <%= f.submit 'Add Category', class:'btn-category' %>
+</div>
```

Comment:
### Optional
- [ ] It would be better if we position the button at the bottom of the page, this is marked as optional because the previous code reviewer did not make comments about that
![Screenshot from 2023-06-03 16-30-18](https://github.com/The-Normal-One1/budget_app/assets/46388113/53a2fb71-8b49-4113-ab79-df24f86dae37)
![Screenshot from 2023-06-03 16-29-49](https://github.com/The-Normal-One1/budget_app/assets/46388113/85199fcf-4324-467a-80b2-98212f242baf)